src/components/App.js

```diff
@@ -72,11 +59,13 @@ class App extends Component {
       LinkedInLogin.getProfile();
     });
 
-    DeviceEventEmitter.addListener('linkedinGetRequest', (data) => {
-      const data = JSON.parse(data.data);
-      if (data) {
-        console.log(JSON.stringify(data));
-        console.log('data');
+    DeviceEventEmitter.addListener('linkedinGetRequest', (resData) => {
+      const data = JSON.parse(resData.data);
+      if (data.values) {
```

Comment:
Is this right? Is the image the only thing that's stored in data.values? Even if that's the case, I wonder if there's a more explicit way to check for images. OR, the "reject" statement below should just read "resData is empty".